packages/cactus-plugin-satp-hermes/src/main/typescript/plugin-satp-hermes-gateway.ts

```diff
@@ -266,32 +263,32 @@ export class SATPGateway implements IPluginWebService, ICactusPlugin {
       pluginOptions.keyPair = Secp256k1Keys.generateKeyPairsBuffer();
     }
 
-    const id = uuidv4();
     if (!pluginOptions.gid) {
       pluginOptions.gid = {
-        id: id,
+        id: uuidv4(),
         pubKey: bufArray2HexStr(pluginOptions.keyPair.publicKey),
-        name: id,
+        name: uuidv4(),
         version: [
           {
-            Core: "v02",
-            Architecture: "v02",
-            Crash: "v02",
+            Core: SATP_VERSION,
```

Comment:
This cant be like, these are 3 kind of versions, they are not all the same, if they were we only whould have 1 version variable, please add 3 diferent variables for the versions. SATP_CORE_VERSION, SATP_ARCHITETURE_VERSION, SATP_CRASH_VERSION

Reply:
sure, I'll add those